### DISCUSSION TITLE
Can "DNS" or "DDNS" be supported in regions where fixed IP cannot be obtained?

### DISCUSSION THREAD

**Discussion by ChaiWenyu1983** [filecoin-project/lotus#8937] [Ideas] [answered]
Fixed IP can not be owned in some places, or the cost of ownership is very high, but they can easily obtain fixed "DDNS". Can an interstellar file system support this scenario? This allows more people to participate in the system.

**Comment by rjan90** [comment, marked as answer]
Hey @ChaiWenyu1983! Yes, I know that a lot of storage providers and node operators are currently using and running their operations with a DDNS based multiaddresse 😄! So that is possible!
